src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToGenerateGlobalUsings_DotNet.cs

```diff
@@ -103,10 +105,21 @@ public void It_can_generate_custom_imports()
         {
             var tfm = "net6.0";
             var testProject = CreateTestProject(tfm);
-            testProject.AdditionalProperties["DisableImplicitNamespaceImports_DotNet"] = "true";
-            testProject.AdditionalItems["Import"] = new Dictionary<string, string> { ["Include"] = "CustomNamespace" };
+            testProject.ProjectChanges.Add(projectXml =>
+            {
+                var ns = projectXml.Root.Name.Namespace;
+                var itemGroup = new XElement(ns + "ItemGroup");
+                projectXml.Root.Add(XElement.Parse(
+@"<ItemGroup>
+    <Using Include=""CustomNamespace"" />
+    <Using Include=""TestStaticNamespace"" Static=""True"" />
+    <Using Include=""System.Biology"" Alias=""AppliedChemistry"" />
+</ItemGroup>"));
+            });
+
+            testProject.AdditionalItems["Using"] = new Dictionary<string, string> { ["Include"] = "CustomNamespace" };
```

Comment:
FYI: #19596 changes the API for adding additional items to a TestProject, so if that PR merges first this will need to be updated.

Also, it looks like you're including a `Using` for `CustomNamespace` twice, is that intentional?